src/Text/Pandoc/ImageSize.hs

```diff
@@ -271,6 +280,23 @@ gifSize img = do
                           }
        _             -> Nothing -- "GIF parse error"
 
+svgSize :: ByteString -> Maybe ImageSize
+svgSize img = do
+  doc <- Xml.parseXMLDoc $ UTF8.toString img
+  let opts = def --TODO: use proper opts instead of def, which simply contains dpi=72
```

Comment:
We might as well do it right, even if it requires an interface change.
2.0 is a great opportunity to break things that need breaking.